packages/fsfoundation/src/ExtractPropertyNames.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Extracts the property names from a given object whose values are assignable to a given type
+ *
+ * Example:
+ * interface MyObject {
+ *  a: string;
+ *  b: string;
+ *  c: () => void;
+ *  d: number;
```

Comment:
Shouldn't these have one more space to be two spaces from where `interface` is indented to?